linode/manager
# M3-5804: Update All No Stats Available States

## Description

- Updates _No Stats Available_ states across Cloud Manager ☁️
  - **Linodes**
    -  No stats calls will be made to the API until 7 minutes after creation
    -  Polls for new stats every 30 seconds
  - **NodeBalancers**
    - No stats calls will be made to the API until 5 minutes after creation
    - Polls for new stats every 20 seconds

![Screen Shot 2022-04-20 at 5 37 17 PM](https://user-images.githubusercontent.com/6440455/164327283-67705824-cc98-4fa3-9c85-9cd339dcc4b1.jpg)

![Screen Shot 2022-04-20 at 5 38 38 PM](https://user-images.githubusercontent.com/6440455/164327288-6df50f36-bf0b-4e1e-9467-a862a6c16695.jpg)

## How to test

- Create a NodeBalancer and check the stats unavailable state
  - Verify the new stats unavailable state remains until stats are ready
- Create a Linode and check the stats unavailable state
  - Verify the new stats unavailable state remains until stats are ready
  - Compare functionality to production
  - Ensure Date range select still works properly
